asgiref/sync.py

```diff
@@ -82,7 +82,9 @@ def __get__(self, parent, objtype):
         """
         Include self for methods
         """
-        return functools.partial(self.__call__, parent)
+        func = functools.partial(self.__call__, parent)
+        func.__qualname__ = self.awaitable.__qualname__
```

Comment:
I'm not sure about the design of `partial`, but it might be better to guard this with `if not hasattr(func, "__qualname__")` or similar, in case a future version of Python fixes/changes it.

Reply:
Thanks @davidism, see [other comment](https://github.com/django/asgiref/pull/104#issuecomment-509353970) for new/better/recommended approach.